### PR TITLE
Added English (UK)

### PR DESCRIPTION
**Pull request by GreenTurtwig**
Also fixed a small error in US

### CODE REVIEW

File: src/_locales/_provider.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const localeLinks = {
   'en-AU': 'en-US',
   'en-BZ': 'en-US',
   'en-CA': 'en-US',
-  'en-GB': 'en-US',
```
Comment (MarshallOfSound) [inline src/_locales/_provider.js]: You can completely delete this line now

Comment (GreenTurtwig) [inline src/_locales/_provider.js]: Completely forgot about that. :-P

**Comment by MarshallOfSound** [comment]
In general I don't like this much duplicate, pretty much all the strings are the same as `en-US` which makes adding new strings just that much more annoying.

I would suggest instead modifying `TranslationProvider` to accept js files aswell and simply use `Object.assign()` to override the few object properties you want to change for UK

**Comment by jostrander** [comment]
This PR is stale, closing.

**Pull request closed without merging**